agent/installer/cli.go

```diff
@@ -8,11 +8,12 @@ import (
 	"fmt"
 	"github.com/go-logr/logr"
 	"k8s.io/klog/klogr"
+	"os"
+	"text/tabwriter"
 )
 
 var (
-	listSupportedFlag    = flag.Bool("list-supported", false, "List all supported OS and Kubernetes versions")
-	listBundlesFlag      = flag.Bool("list-bundles", false, "List the BYOH Bundle names for all supported OS and Kubernetes versions")
+	listSupportedFlag    = flag.Bool("list-supported", false, "List all supported OS, Kubernetes versions and BYOH Bundle names")
```

Comment:
Just a question, maybe I'm wrong :) 
Should we add the list support bundles function in the management cluster side?

If I'm an end user, after I install the byoh provider , then I will expect to check what bundles do we have in the repo on the management cluster side instead of login to the remote byoh host. 

Then I can decide which version of kubernest/ bundle I would like to use to create the cluster on byoh host.

Reply:
Makes sense. This is just a CLI of the installer lib that is used by the host agent. So, the CP can retrieve the same info by the lib directly. The exact workflow for that is outside installer lib charter.

Reply:
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/pull/207#discussion_r743457774